src/schema/me/bidder_position_mutation.js

```diff
@@ -40,5 +112,22 @@ export const BidderPositionMutation = mutationWithClientMutationId({
       return new Error("You need to be signed in to perform this action")
     }
     return createBidderPositionLoader({ sale_id, artwork_id, max_bid_amount_cents })
+      .then(p => { return { status: "SUCCESS", position: p } })
+      .catch(e => {
+        const errorSplit = e.message.split(" - ")
+        const errorObject = errorSplit.length > 1 ? JSON.parse(e.message.split(" - ")[1]) : null
+        if (errorObject && errorObject.type === "param_error") {
+          const error = biddingErrors.find(d => errorObject.message.trim().startsWith(d.gravity_key)) ||
+            errorObject.ERROR_UNKNOWN
+          const liveAuctionUrl = `${PREDICTION_ENDPOINT}/${sale_id}`
```

Comment:
A note that when generating the URL to a live auction, we generally [care whether or not you are logged in](https://github.com/artsy/force/blob/master/src/utils/domain/auctions/urls.js#L11-L19), but I think in this case this URL is fine since you can't get to the bid screen unless you're logged in.

Reply:
Not sure but I think we throw error when request is not coming from a logged in user with 
```
 if (!accessToken) {
      return new Error("You need to be signed in to perform this action")
    }
```
above.

Reply:
You're right! Definitely covered then 👍